### ISSUE TITLE
Object Destructing Import Not Working

### ISSUE THREAD

**Issue by 0xcaff** [skovhus/css-modules-flow-types#10]
I use css modules as follows:

```js
import {
  container as containerClass,
  heartContainer as heartClass,
  fill as fillClass,
  liked as likedClass,
} from './Like.css';
```

When I try doing this with generated flow types, I get the following error:

![image](https://user-images.githubusercontent.com/3277097/34636849-d773d978-f277-11e7-97c0-d0efb8b40adf.png)

> This module only has a default export. Did you mean `import container from ...`?

Here's the generated file:

```js
// @flow
/* This file is automatically generated by css-modules-flow-types */
declare module.exports: {|
  +'container': string;
  +'fill': string;
  +'heartContainer': string;
  +'like-in': string;
  +'like-out': string;
  +'liked': string;
|};
```

It seems that this is caused because flow doesn't understand that importing fields of an object is the same as destructing it. Changing the content to the following form works:

```js
// @flow
/* This file is automatically generated by css-modules-flow-types */
declare export const content: string;
declare export const footer: string;
declare export const grid: string;
declare export const sidebar: string;
```

But this begs the question of how to declare global exports for fields with dashes in the name like `like-in` and `like-out`.

**Comment by skovhus** [comment]
Thanks for reporting this. 

Wondering why Flow doesn't understand importing fields is the same as destructing it. That seems like that is the bug were should solve. 🤔 

### Unrelated sidenote: 

I'm curious why you map your styles object... Would like to understand the use case. 

To me it seems like some unnecessary bookkeeping:

```javascript
import {
  foo as fooClass,
  bar as barClass,
  baz as bazClass,
} from './Text.css';

const usingStyle1 = fooClass
const usingStyle2 = barClass
const usingStyle3 = bazClass

// vs 

import styleClasses from './Text.css';

const usingStyle1 = styleClasses.foo
const usingStyle2 = styleClasses.bar
const usingStyle3 = styleClasses.baz
```

**Comment by 0xcaff** [comment]
I started this this project using just a text editor and eslint. The object destructing makes it easier to see what's being used from a css file and eslint knows when classes aren't used. For example, eslint knows that

```js
import { unused as unusedClass } from './file.css'
```

is unused. It doesn't know anything about.

```js
import style from './file.css'
// style.unused is unused but no eslint error occurs.
```

I've switched to an IDE and `import style from './file.css'` because the tooling for this form is better (this tool, ide understands it).
